src/components/draggable-list/DraggableList.tsx

```diff
@@ -22,6 +22,7 @@ const DraggableList = ({ children, className, listId, onDragEnd }: DraggableList
                 {droppableProvided => (
                     <div ref={droppableProvided.innerRef} className={className}>
                         {children}
+                        {droppableProvided.placeholder}
```

Comment:
This is to avoid warning in the console, since version 10 (I believe), Droppable need to use placeholder.